### DISCUSSION TITLE
Display Pod name instead of task Name in Kubeflow Pipeline UI V2

### DISCUSSION THREAD

**Discussion by EmilioCC** [kubeflow/pipelines#10723] [General]
In Kubeflow Pipeline UI V1, when we click on a task we can see the pod name for the executing task. If we click on the task in the V2, we see the name of the task.

We've been using the fact that we could see the name of the pod for the underlying running task in order to facilitate debugging whenever we had to look at the pod's log using Kubectl or exploring logs using GCP's Log Explorer. This is not specific to running Kubeflow Pipelines on Kubernetes in GCP.

<img width="1301" alt="image" src="https://github.com/kubeflow/pipelines/assets/6756551/8fb7b51c-d520-4314-a5fa-ae4d3b8061d7">

**Comment by rimolive** [comment]
With KFPv2, things changed a bit and now there's heavy dependency on Google's ml-metadata project. Most of the information provided by kfpv1 now is returned by ml-metadata.

I think we can agree that this should be transparent to the users and UI should not drastically change, but I believe there are other places that must be changed as well.
